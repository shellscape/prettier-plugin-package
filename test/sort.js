/* eslint-disable import/no-unresolved, sort-keys */

import { test, expect } from 'vitest';
import prettier from 'prettier';

import plugin from '../lib/index.js';

test('props', async () => {
  const options = {
    filepath: 'package.json',
    parser: 'json-stringify',
    plugins: [plugin],
  };
  const fixture = {
    keywords: ['package', 'package.json', 'plugin', 'prettier'],
    version: '0.1.1',
  };

  const input = JSON.stringify(fixture, null, 2);
  const output = await prettier.format(input, options);

  expect(output).toMatchSnapshot();
});
