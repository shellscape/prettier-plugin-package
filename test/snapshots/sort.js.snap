// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`props 1`] = `
"{
  "version": "0.1.1",
  "keywords": [
    "package",
    "package.json",
    "plugin",
    "prettier"
  ]
}
"
`;
